views/replAuth.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en" class="login">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Best Posts!</title>
+		<link
+			rel="stylesheet"
+			type="text/css"
+			href="https://Replit-2020.whippingdot.repl.co/static/style.css"
+		/>
+		<link
+			rel="shortcut icon"
+			href="https://Replit-2020.whippingdot.repl.co/static/favicon_but_better.ico"
+			type="image/x-icon"
+		/>
+	</head>
+	<body class="center">
+		<h1>Log In</h1>
+		<h2>Log in using your repl.it account!</h2>
+		<script
+			authed="alert('You Logged In')"
```

Comment:
yes. it needs to refresh, and just throw in an if statement in pugjs or something. I'll get round to converting.

Reply:
Read my and @AmazingMech2418 's comments. We might ditch this page altogether and make it just a button on the suggestions page. Sorry...

Reply:
@firefish111 I like pugjs and all but it is confusing. To keep it simple let us not use pugjs here.

Reply:
pugjs is YAML-like, so if you know YAML it should be easy to use.
YAML (YAML Ain't a Markup Language)  basically is fancy indent-reliant json.
For example: (from google):
```yml
---
- hosts: webservers

  vars:
    http_port: 80
    max_clients: 200

  remote_user: root

  tasks:
  - name: ensure apache is at the latest version
    yum:
      name: httpd
      state: latest

  - name: write the apache config file
    template:
      src: /srv/httpd.j2
      dest: /etc/httpd.conf
    notify:
    - restart apache

  - name: ensure apache is running
    service:
      name: httpd
      state: started

  handlers:
    - name: restart apache
      service:
        name: httpd
        state: restarted
```
It is heavily indent-reliant, much like Python and Nim. That means, one incorrect indent, and your program might not behave as expected.

Reply:
K...But most of us don't use pug. I can create a team under web dev for pugjs if you want but this project will remain strictly JS, CSS, HTML, and Node.